src/TimePicker/TimePicker.tsx

```diff
@@ -41,6 +43,21 @@ const isValidHoursInTimeFormat = (
   timeFormat: TimePicker.TimeFormat
 ): hours is number => (timeFormat === H24 ? Hour.is(hours) : Hour12.is(hours));
 
+const getComponents = (
+  timeFormatter?: TimePicker.TimeFormatter
+): SelectNS.Props<TimeDropdownOption>["components"] => {
+  return timeFormatter
+    ? {
+        Option: props => (
+          <components.Option {...props}>
+            {timeFormatter(props.data.time) || null}
+          </components.Option>
+        ),
+        SingleValue: props => timeFormatter(props.data.time) || null
```

Comment:
is it ok to pass ` JSX.Element` as `SingleValue`?
also, same comment as above: is `|| null` necessary?

Reply:
the first answer is "yes", the second is "not and I have no clue why I put this `||`"...